In [6]:
print("Hello AI Word")

Hello AI Word


# Printing in Python

The following code prints **Hello AI Word** on the screen.

```python
print("Hello AI Word")

In [1]:
name ="Ahmad"
roll_no = 19
cgpa = 3.96

print(" name : " , name)
print(" roll_no : " , roll_no)
print(" cgpa : " , cgpa)

 name :  Ahmad
 roll_no :  19
 cgpa :  3.96


# Student Information in Python

The following Python code stores a student's **name**, **roll number**, and **CGPA** in variables.  
Then it prints them using the `print()` function.



In [4]:
for val in range( 1 ,11):
    print(val)
print()

#after modify code   
for val in range( 1 ,11):
    if val % 2==0:
        print(val)


1
2
3
4
5
6
7
8
9
10

2
4
6
8
10


# Printing Numbers with Python Loops

We will use a `for` loop to print numbers from **1 to 10**.  
Later, we will modify the code to print only **even numbers**.


In [5]:
def factorial(n):
    f = 1
    for val in range(1 , n+1):
        f *= val

    return f


fact=factorial(5)
print("The Factorial of 5 is -> ", fact)

def factorial(n):
    f = 1
    for val in range(1 , n+1):
        f *= val

    return f

x=int(input("Enter Number For Factorial"))
fact=factorial(x)
print(fact)

The Factorial of 5 is ->  120


Enter Number For Factorial 5


120


# Factorial Program in Python

The **factorial** of a number `n` (written as `n!`) is the product of all positive integers less than or equal to `n`.  
For example:  
- 5! = 5 × 4 × 3 × 2 × 1 = 120  

We can calculate factorial in Python using a **for loop**.

In [ ]:
marks=int(input("Enter your marks ->  "))

if marks >= 80 :
    print("Grade A")
elif marks >= 60:
    print("Grade B")
elif marks>= 50:
    print("Grade c")
else:
    print("Fail")

# Grading System in Python

We use **if-elif-else** conditions to assign grades based on marks.

---

## 🔹 Steps (List Form)

1. Take input from the user using `input()` and convert it into an integer:
   ```python
   marks = int(input("Enter your marks ->  "))
2. It print `Grade` base on condition.